src/lib/containers/AccessApprovalsTable.tsx

```diff
@@ -0,0 +1,207 @@
+  const [arList, setArList] = useState<AccessApprovalSearchResult[]>()
+
+  const getApprovals = async () => {
+    try {
+      const searchRequest: AccessApprovalSearchRequest = {
+        accessorId,
+        accessRequirementId,
+      }
+      const approvalResponse = await searchAccessApprovals(
+        searchRequest,
+        accessToken,
+      )
+      setSearchResult(approvalResponse)
+      setArList(approvalResponse.results)
+    } catch (err: any) {
+      displayToast(err.reason as string, 'danger')
+    }
+  }
+
+  const onLoadMore = async () => {
+    setIsLoading(true)
+    try {
+      const searchRequest: AccessApprovalSearchRequest = {
+        accessorId,
+        accessRequirementId,
+        nextPageToken: searchResult?.nextPageToken,
+      }
+      const approvalResponse = await searchAccessApprovals(
+        searchRequest,
+        accessToken,
+      )
+      setSearchResult(approvalResponse)
+      if (arList) {
+        setArList([...arList, ...approvalResponse.results])
+      }
+    } catch (err: any) {
+      displayToast(err.reason as string, 'danger')
+    } finally {
+      setIsLoading(false)
+    }
+  }
+
+  useEffect(() => {
+    getApprovals()
+  }, [accessToken, accessorId, accessRequirementId])
+
+  const sortColumn = async (
+    field: AccessApprovalSortField,
+    isDescending: boolean,
+  ) => {
+    try {
+      const direction = isDescending
+        ? ('DESC' as SortDirection)
+        : ('ASC' as SortDirection)
+      const sort: AccessApprovalSearchSort[] = [
+        {
+          field,
+          direction,
+        },
+      ]
+      const searchRequest: AccessApprovalSearchRequest = {
+        accessorId,
+        accessRequirementId,
+        sort,
+      }
+      const sortRequest = await searchAccessApprovals(
+        searchRequest,
+        accessToken,
+      )
+      setSearchResult(sortRequest)
+      if (arList) {
+        setArList([...arList, ...sortRequest.results])
+      }
+      setIsDescending(!isDescending)
+    } catch (err) {
+      console.error(err)
+    }
+  }
+  return (
+    <div className="AccessApprovalsTable">
+      <Typography style={{ padding: '12px' }} variant="headline3">
+        Status in Access Requirements
+      </Typography>
+      <Table striped borderless bordered={false}>
+        <thead className="access-approval-header">
+          <tr>
+            <th className="thead-cell remove-border">AR ID</th>
+            <th className="thead-cell remove-border">
+              Access Requirement Name
+            </th>
+            <th className="thead-cell remove-border">Submitter</th>
+            <th className="thead-cell remove-border">Status</th>
+            <th className="thead-cell remove-border button-cell">
+              Modified Date
+              <button
+                style={{ float: 'right' }}
+                onClick={() =>
+                  sortColumn(
+                    'MODIFIED_ON' as AccessApprovalSortField,
```

Comment:
Use the value instead of casting
```suggestion
                    AccessApprovalSortField.MODIFIED_ON,
```